src/Microsoft.AspNetCore.Razor.Language/Syntax/ArrayElement.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal struct ArrayElement<T>
```

Comment:
Copy paste.

Reply:
hmmm, where from? We can't just copy arbitrary code. You need to discuss with @Eilon

Reply:
From Roslyn
https://github.com/dotnet/roslyn/blob/cfdd6068c7e8b63c5ad1e71eaddda61962b51b9a/src/Compilers/Core/Portable/Collections/ArrayElement.cs#L8